spec/features/review_spec.rb

```diff
@@ -8,7 +8,7 @@
     login(user.email)
   end
 
-  context 'When there are cards for review' do
+  context 'When there are cards_mailer for review' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.